### DISCUSSION TITLE
Sorting environments by name

### DISCUSSION THREAD

**Discussion by tretail** [community/community#32556] [Actions]
Hello everyone,

I think it would be great to sort environments alphabetically by name into the Environments settings' page (https://github.com/*my-org*/*my-project*/settings/environments).

![Screenshot 2022-09-10 at 14 43 59](https://user-images.githubusercontent.com/9505079/189483919-92c2d59f-5b20-4430-ac8e-d27da33a664b.png)

I actually use this to inject secrets depending of which application I want to deploy on which environment, and though may have a large amount of environments. At the moment, the last created environment appears at the top of the list.

Have a nice day,

**Comment by dmilburn** [comment]
Hi @tretail, thanks for reaching out and for your feedback. This functionality is owned by the Actions team and not Codespaces. Could you please change the category of your discussion to "Actions and Packages" so the right people see it?
